arm/Microsoft.EventHub/namespaces/deploy.bicep

```
@@ -235,7 +235,7 @@ module eventHubNamespace_eventHubs 'eventhubs/deploy.bicep' = [for (eventHub, in
     captureDescriptionSizeLimitInBytes: contains(eventHub, 'captureDescriptionSizeLimitInBytes') ? eventHub.captureDescriptionSizeLimitInBytes : 314572800
     captureDescriptionSkipEmptyArchives: contains(eventHub, 'captureDescriptionSkipEmptyArchives') ? eventHub.captureDescriptionSkipEmptyArchives : false
     consumerGroups: contains(eventHub, 'consumerGroups') ? eventHub.consumerGroups : []
-    lock: contains(eventHub, 'lock') ? eventHub.lock : 'NotSpecified'
+    lock: contains(eventHub, 'lock') ? eventHub.lock : lock
```

As its a child of the eventhub, it would automatically inherit the lock. Suggest this default go to `''` but provide a lock is explicitly added.
```suggestion
    lock: contains(eventHub, 'lock') ? eventHub.lock : ''
```

---

done